Vertexvis/vertex-web-sdk
# [PLAT-2787] Hit Result indicator

## Summary

Adds a new `<vertex-viewer-hit-result-indicator>` element that can be used to display visuals for the position, normal, and computed plane of a hit result. This component leverages WebGL to draw the indicator, reusing some of the types added with the `<vertex-viewer-transform-widget>`, and makes updates to extract some of the shared behavior between the two.

https://user-images.githubusercontent.com/5252480/236006708-97e1eb69-8979-4ead-968c-ac7d81cf4ce0.mov

## Test Plan

- Verify that the transform widget continues to work as expected
  - Setting position, clearing position, etc
- Place multiple `<vertex-viewer-hit-result-indicator>`s using data from hit results, and verify the position, normal, and computed plane visually match the data in the hit result

## Release Notes

N/A

## Possible Regressions

Transform widget

## Dependencies

N/A
